sdk/storage/Azure.Storage.Files/src/FileClient.cs

```diff
@@ -1682,6 +1682,167 @@ public virtual FileClient WithSnapshot(string shareSnapshot)
         }
         #endregion UploadRange
 
+        #region UploadRangeFromUrl
+        /// <summary>
+        /// The <see cref="UploadRangeFromUrl"/> operation writes a range from an Azure File to another Azure file. 
+        /// This API is supported only for version 2019-02-02 and higher.
+        /// </summary>
+        /// <param name="sourceUri">
+        /// Required. Specifies the URL of the source file, up to 2 KB in length.
+        /// If source is an Azure blob or Azure file, it must either be public or must be authenticated via a 
+        /// shared access signature. If the source is public, no authentication is required to perform the operation.
+        /// </param>
+        /// <param name="range">
+        /// Specifies the range of bytes to be written. Both the start and end of the range must be specified.
+        /// </param>
+        /// <param name="sourceRange">
+        /// Specifies the range of bytes to be written from. Both the start and end of the range must be specified.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate
+        /// notifications that the operation should be cancelled.
+        /// </param>
+        /// <returns>
+        /// A <see cref="Response{StorageFileUploadInfo}"/> describing the
+        /// state of the file.
+        /// </returns>
+        /// <remarks>
+        /// A <see cref="StorageRequestFailedException"/> will be thrown if
+        /// a failure occurs.
+        /// </remarks>
+        public virtual Response<StorageFileUploadInfo> UploadRangeFromUrl(
```

Comment:
This should be named `UploadRangeFromUri` because we use Uri instead of Url for all our other similar names

Reply:
fixed.